aldjemy/table.py

```diff
@@ -88,7 +88,10 @@ def generate_tables(metadata):
                     or (f.many_to_one and f.related_model)
             ]
         for field, parent_model in model_fields:
-            if field not in model._meta.virtual_fields:
+            private_fields = (model._meta.private_fields  # >= Django 1.10
```

Comment:
* move it from `for` cycle
* it is a bad place to use `do if else` construct, I suggest to use good old `if: priv = bla else: priv = qwert`

Reply:
Fine on the first point (fixed now), I don't know what you mean on the second point about it being a 'bad place' to use `if else`.